*** ./smali/android/database/AbstractCursor.smali	2015-09-08 15:31:32.195784624 +0800
--- /home/baohailong/patchrom/0901la/temp/new_smali/framework.jar.out/./smali/android/database/AbstractCursor.smali	2015-09-08 15:32:58.824257440 +0800
***************
*** 825,830 ****
--- 825,839 ----
      goto :goto_0
  
      :cond_4
+     invoke-static {p0, p1}, Landroid/database/AbstractCursorInjector;->checkPosition(Landroid/database/AbstractCursor;I)Z
+ 
+     move-result v2
+ 
+     if-nez v2, :cond_miui_0
+ 
+     return v2
+ 
+     :cond_miui_0
      iput p1, p0, Landroid/database/AbstractCursor;->mPos:I
  
      iget v2, p0, Landroid/database/AbstractCursor;->mRowIdColumnIndex:I
